static/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+
+<link rel="stylesheet" href="../client/vendor/assets/fonts/fonts.css" media="screen">
+<link rel="stylesheet" href="../client/node_modules/normalize.css/normalize.css" media="screen">
+<link rel="stylesheet" href="./main.css" media="screen" charset="utf-8">
+
+<meta name="apple-mobile-web-app-capable" content="yes">
+<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
+
+<div role="application">
+  <main>
+
+    <header>
+        <a href="https://cozy.io" class="cozy">
+          <img src="assets/illustration-welcome.svg" alt="Welcome to Cozy cloud" />
+        </a>
+
+        <h1>Welcome Michel</h1>
+        <a href="#">You're not Michel?</a>
+    </header>
+
+    <div role="region">
+      <p>
+        Congratulation, thanks to you, your personal data are in a safe place, in your Cozy.
+      </p>
+    </div>
+
+    <footer>
+      <div class="controls">
+        <a href="#" role="button">next</a>
```

Comment:
I use it here because I'm not sure at this time how the next button will be handled in code. As steps have their own URIs, and as long the welcome screen doesn't contain a <form> to submit, this one is probably more a link to the next step rather than a button that doesn't send anything.